### ISSUE TITLE
getUserLogin() fails

### ISSUE THREAD

**Issue by calve** [calve/car-01-ftp#1]
Unknow failure on `getUserLogin` using `anonymous`

```
run:
     [java] Welcome in this ftpd java
     [java] Now listening on port 1025
     [java] New incoming connection
     [java]  --> 220 ready
     [java] 
     [java]  <-- USER anonymous
     [java] set user to (%s)
     [java] anonymous
     [java]  --> 331 Username ok, send password.
     [java] 
     [java]  <-- PASS plop
     [java]  --> 230 User loged in, proceed
     [java] 
     [java] Exception in thread "Thread-0" java.lang.NullPointerException
     [java]     at ftpd.FtpRequest.getUserByLogin(Unknown Source)
     [java]     at ftpd.FtpRequest.processPass(Unknown Source)
     [java]     at ftpd.FtpRequest.processRequest(Unknown Source)
     [java]     at ftpd.FtpRequest.run(Unknown Source)
```

**Comment by calve** [comment]
Fixed in 3253976f6e101108b1e97f1a0f7713ce3c15c217

**Comment by calve** [comment]
Fixed in 3253976f6e101108b1e97f1a0f7713ce3c15c217
